lib/local-cluster/local-cluster.cabal

```
@@ -0,0 +1,78 @@
+cabal-version:      3.0
+name:               local-cluster
+version:            0.1.0.0
+synopsis:           Local cluster of cardano nodes
+-- description:
+homepage:           https://github.com/cardanofoundation/cardano-wallet
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Hal, Cardano Foundation
+maintainer:         hal@cardanofoundation.org
+-- copyright:
+category:           Web
+build-type:         Simple
+extra-doc-files:    CHANGELOG.md
+-- extra-source-files:
+
+common language
+  default-language:   Haskell2010
+  default-extensions:
+    NoImplicitPrelude
+    OverloadedStrings
+
+common opts-lib
+  ghc-options: -Wall -Wcompat -Wredundant-constraints
+
+  if flag(release)
+    ghc-options: -O2 -Werror
+
+common opts-exe
+  ghc-options: -threaded -rtsopts -Wall -Wredundant-constraints
+
+  if flag(release)
+    ghc-options: -O2 -Werror
+
+flag release
+  description: Enable optimization and `-Werror`
+  default:     False
+  manual:      True
+
+executable local-cluster
+    import:         language, opts-exe
+    main-is:          Main.hs
+    ghc-options:     -Wincomplete-uni-patterns -Wincomplete-record-updates
+    build-depends:
+        base
+        , local-cluster
+
+
+    hs-source-dirs:   exe
+    default-language: Haskell2010
+
+library
+  import:          language, opts-lib
+  hs-source-dirs:  lib
+  ghc-options:     -Wincomplete-uni-patterns -Wincomplete-record-updates
+
+
+  exposed-modules:
+    Faucet
+    Service
+  build-depends:
+    base
+    , cardano-wallet
+    , bytestring
+    , text
+    , cardano-wallet-primitive
+    , unliftio
+    , memory
+    , base58-bytestring
+    , cardano-addresses
+    , filepath
+    , directory
+    , text-class
+    , contra-tracer
+    , iohk-monitoring
+    , cardano-wallet-launcher
+    , lobemo-backend-ekg
+    , cardano-wallet:cardano-wallet-api-http
```

```suggestion
    , cardano-wallet:cardano-wallet-api-http

```

Git likes newlines at the end of a file. 🤷